promise/promiseTimeout.ts

```diff
@@ -0,0 +1,3 @@
+(() => {
+    // Implemenet me 💪
+})();
```

Comment:
Why are you forcing an IIFE? 🤔

Reply:
<img src="https://media0.giphy.com/media/MaUEWsln4VgJYjp5OZ/giphy.gif"/>